### ISSUE TITLE
Silent fail of installation on Ubuntu 20.04 LTS with python 3.8.10 and node 18.15.0

### ISSUE THREAD

**Issue by padit1337** [cocktailpeanut/dalai#349]
I have a virtually brand new ubuntu 20.04 (only installations are VS code and spotify) my Python is 3.8.10 and my node is 18.15.0. 
I get this: (removed my name with xxx)
`xxx@xxx-xps-13b:~$ npx dalai llama install 7B
Need to install the following packages:
  dalai@0.3.1
Ok to proceed? (y) y
`
I think I did everything as instructed and installed all necessary software. Am I missing something?

**Comment by bendeguzszkalka** [comment]
No, this issue was submitted before, I have this too. It does not work for some reason. You could try upgrading to python 3.10 or later, but it did not fix it for me.

**Comment by jimaldon** [comment]
same, looks like this repo hasn't been updated in 2 weeks. Plus, llama-dl has a DMCA takedown

**Comment by luck7x** [comment]
> I have a virtually brand new ubuntu 20.04 (only installations are VS code and spotify) my Python is 3.8.10 and my node is 18.15.0. I get this: (removed my name with xxx) I think I did everything as instructed and installed all necessary software. Am I missing something?`xxx@xxx-xps-13b:~$ npx dalai llama install 7B Need to install the following packages: dalai@0.3.1 Ok to proceed? (y) y `

have you fixed this problem? I have the same problem as you, our environment is the same

**Comment by AntonOfTheWoods** [comment]
Same. I'm not sure anything is working anymore, as the alpaca versions (which appear to be downloaded from huggingface) fail silently for me too.

**Comment by bendeguzszkalka** [comment]
> same, looks like this repo hasn't been updated in 2 weeks. Plus, llama-dl has a DMCA takedown

Probably that is the issue. Can we find a way to torrent the files from another source? How should the files be saved and how should they be saved?

**Comment by w34sel** [comment]
sudo  npx dalai llama install 7B

or

change ownership: sudo chown -R $(whoami) ~/.npm 

~~should fix the problem~~

fixed it for me

**Comment by bendeguzszkalka** [comment]
> sudo npx dalai llama install 7B
> 
> or
> 
> change ownership: sudo chown -R $(whoami) ~/.npm
> 
> should fix the problem

it still does not work
